tempto-core/src/main/java/io/prestodb/tempto/internal/convention/tabledefinitions/ConventionTableDefinitionDescriptor.java

```diff
@@ -83,6 +87,13 @@ public ConventionTableDefinitionDescriptor(Path ddlFile)
         this.ddlFile = ddlFile;
 
         Path dataFile = changeExtension(ddlFile, "data");
+        Path dataGeneratorFile = changeExtension(ddlFile, "data-generator");
+        if (exists(dataGeneratorFile)) {
+            checkArgument(!exists(dataFile), "Expected no data file when data-generator file is present");
+            makeExecutable(dataGeneratorFile);
```

Comment:
ouch. This should be read-only.

Reply:
It is a temporary copy of read only files